pages/k8s/1.16/components.md

```diff
@@ -0,0 +1,329 @@
+---
+wrapper_template: kubernetes/docs/base_docs.html
+markdown_includes:
+  nav: kubernetes/docs/shared/_side-navigation.md
+context:
+  title: Components of Charmed Kubernetes 1.16
+  description: Detailed description of Charmed Kubernetes release
+keywords: component, charms, versions, release
+tags:
+    - reference
+sidebar: k8smain-sidebar
+permalink: 1.16/components.html
+layout:
+    - base
+    - ubuntu-com
+toc: false
+bundle_revision: '316'
+bundle_release: '1.16'
+---
+
+<!-- THIS PAGE IS AUTOGENERATED -->
+<!-- If you find an error in this page, it is likely to be in the original
+source of the information - please file a bug rather than editing the text -->
+
+This page details the charms, snaps, images and other components
+which go to make up the 1.16 release of Charmed Kubernetes.
+
+Other information about this release can be found on the following pages:
+
+<a class='p-button--brand' href='/kubernetes/docs'> Documentation </a>
+<a class='p-button--brand' href='/kubernetes/docs/1.16/release-notes'>Release notes </a>
+<a class='p-button--brand' href='/kubernetes/docs/1.16/upgrading'>Upgrade notes </a>
+<a class='p-button--brand' href='https://bugs.launchpad.net/charmed-kubernetes'>Bugs </a>
+<a class='p-button--brand' href='https://github.com/charmed-kubernetes/bundle'>Source </a>
```

Comment:
`Source` here makes me think i'll find all the source that goes into CK, but it's only linking to the bundle repo.  I think this will confuse readers because that repo is about building a CK bundle vs finding CK component source.

Maybe better to link to https://github.com/charmed-kubernetes instead (though this has the problem of dumping readers into an uncategorized mess of all our repos).

Reply:
Well, of course the problem is that there is no single source. I feel the bundle repo is appropriate because that is in essence what is being installed - it is the bundle which decides what the other sources are.

Linking to just the org means people are faced with a large number of repos that aren't really much use to them. Maybe the answer is to add more to the bundle repo readme file to explain the sources it uses? But also, of course, the main components of these are on this same page

Reply:
I see your point, but there's also nowhere in the bundle repo where you would find the 1.16 bundle.yaml -- just a bunch of fragments, overlays, and instructions on running the `bundle` script.

What do you think about pointing this to https://api.jujucharms.com/charmstore/v5/bundle/charmed-kubernetes-410/archive/bundle.yaml (for whatever rev matches 1.16)?

Reply:
In my opinion this should point to https://github.com/charmed-kubernetes.

Reply:
done